examples/stable_diffusion_xl/tools/model_conversion/README.md

```diff
@@ -0,0 +1,97 @@
+# Model Conversion between Mindspore and Torch Inference
+
+We provide scripts of the checkpoint conversion between `.safetensors` and `.ckpt` format. For training or inference in Mindspore, we use the pre-trained weights from hugging face and convert them to `.ckpt` format. Please refer to [the preparation part of GETTING_STARTED.md](https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/GETTING_STARTED.md#convert-pretrained-checkpoint) for details. After finetuning in Mindspore, we can convert the checkpoints back for torch inference as well.
```

Comment:
`.ckpt` can be mindspore or torch checkpoint.  MS checkpoint can also be saved as '.safetensors' format. it's better to state clearly `.safetensors` from torch and `.ckpt` from MS

Reply:
fixed